server/app/models/list.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: lists
+#
+#  id             :integer          not null, primary key
+#  description    :text
+#  items_quantity :integer          default(0), not null
```

Comment:
Probably want to use the automatic counter cache column name… `#{association_name}_count` iirc